test/object-store/migrations.cpp

```diff
@@ -3448,119 +2700,12 @@ TEST_CASE("migration: Manual") {
         auto realm2 = Realm::get_shared_realm(config);
         // will deadlock if it tries to start a write transaction
         REQUIRE_NOTHROW(realm2->update_schema(schema));
-        REQUIRE_THROWS(realm2->update_schema(remove_property(schema, "object", "value")));
+        REQUIRE_THROWS_CONTAINING(realm2->update_schema(remove_property(schema, "object", "value")),
+                                  "Property 'object.value' has been removed.");
     }
 
     SECTION("null migration callback should throw SchemaMismatchException") {
         Schema new_schema = remove_property(schema, "object", "value");
         REQUIRE_THROWS_AS(realm->update_schema(new_schema, 1, nullptr), SchemaMismatchException);
     }
 }
-
-#if REALM_ENABLE_AUTH_TESTS
-
-TEST_CASE("migrations with asymmetric tables") {
```

Comment:
This test case was testing local Realm migrations on a feature that's not supported in local Realms, which isn't something that makes sense to test.